projects/ngneat/transloco/src/lib/transloco.service.ts

```diff
@@ -271,6 +290,20 @@ export class TranslocoService implements OnDestroy {
     return this.load(langName);
   }
 
+  private loadFallbackLang(lang: string): Observable<HashMap<any> | null> {
```

Comment:
There are cases when users want a fallback for when the main language doesn't load, but they don't need it for missing keys or values. I would add a `config.useFallbackForMissingKeys` and only if that's `true`, and we have a fallback you do the merge.

Reply:
I thought about adding an option to enable this behavior. Fixed.